app/controllers/organen/orgaan/mandatarissen.js

```diff
@@ -77,4 +78,25 @@ export default class OrganenMandatarissenController extends Controller {
   toggleActiveOnly() {
     this.activeOnly = !this.activeOnly;
   }
+
+  @action
+  async isSelectedPeriodCurrent() {
```

Comment:
shouldn't this just be a getter?

Reply:
Hm but than I cannot set the tracked properties for it